sparse_strips/vello_sparse_shaders/shaders/render_strips.wgsl

```diff
@@ -33,14 +33,36 @@ const COLOR_SOURCE_BLEND: u32 = 2u;
 
 // Paint types
 const PAINT_TYPE_SOLID: u32 = 0u;  
-const PAINT_TYPE_IMAGE: u32 = 1u;  
+const PAINT_TYPE_IMAGE: u32 = 1u;
+const PAINT_TYPE_LINEAR_GRADIENT: u32 = 2u;
+const PAINT_TYPE_RADIAL_GRADIENT: u32 = 3u;
+const PAINT_TYPE_SWEEP_GRADIENT: u32 = 4u;
+
+// Paint texture index mask (extracts lower 27 bits from paint field).
+const PAINT_TEXTURE_INDEX_MASK: u32 = 0x07FFFFFFu; 
 
 // Image quality
 const IMAGE_QUALITY_LOW = 0u;
 const IMAGE_QUALITY_MEDIUM = 1u;
 const IMAGE_QUALITY_HIGH = 2u;
 
-// Composite modes
+// Gradient types.
+const GRADIENT_TYPE_LINEAR: u32 = 0u;
+const GRADIENT_TYPE_RADIAL: u32 = 1u;
+const GRADIENT_TYPE_SWEEP: u32 = 2u;
+
+// Radial gradient types.
+const RADIAL_GRADIENT_TYPE_STANDARD: u32 = 0u;
+const RADIAL_GRADIENT_TYPE_STRIP: u32 = 1u;
+const RADIAL_GRADIENT_TYPE_FOCAL: u32 = 2u;
+
+// Mathematical constants.
+const PI: f32 = 3.1415926535897932384626433832795028;
+const TWO_PI: f32 = 2.0 * PI;
+// Tolerance for nearly zero comparisons (matching vello_cpu implementation).
+const NEARLY_ZERO_TOLERANCE: f32 = 1.0 / 4096.0;
```

Comment:
Are you able to add a comment [here](https://github.com/taj-p/vello/blob/58b80d660e2fc5aef3bd32b24af3f95e973aab95/sparse_strips/vello_common/src/math.rs#L21) stating that changing that value should also see a change in `render_strips.wgsl`?